apps/antalmanac/src/components/Map/Map.tsx

```diff
@@ -171,16 +171,16 @@ export default function CourseMap() {
     }, []);
 
     useEffect(() => {
-        const updateCalendarEvents = () => {
-            setCalendarEvents(AppStore.getCourseEventsInCalendar());
+        const updateCustomEventMarkers = () => {
+            setCustomEventMarkers(getCustomEventPerBuilding());
         };
 
-        AppStore.on('addedCoursesChange', updateCalendarEvents);
-        AppStore.on('currentScheduleIndexChange', updateCalendarEvents);
+        AppStore.on('customEventsChange', updateCustomEventMarkers);
+        AppStore.on('currentScheduleIndexChange', updateCustomEventMarkers);
 
         return () => {
-            AppStore.removeListener('addedCoursesChange', updateCalendarEvents);
-            AppStore.removeListener('currentScheduleIndexChange', updateCalendarEvents);
+            AppStore.removeListener('customEventsChange', updateCustomEventMarkers);
+            AppStore.removeListener('currentScheduleIndexChange', updateCustomEventMarkers);
```

Comment:
Unless I'm mistaken, I believe these lines should not have been changed. Mixup from duplicate map marker code, which admittedly we maybe shouldn't be handling here in an unrelated PR but uh... why not 🤷‍♂️ 

(from main):
```tsx
    useEffect(() => {
        const updateCalendarEvents = () => {
            setCalendarEvents(AppStore.getCourseEventsInCalendar());
        };

        AppStore.on('addedCoursesChange', updateCalendarEvents);
        AppStore.on('currentScheduleIndexChange', updateCalendarEvents);

        return () => {
            AppStore.removeListener('addedCoursesChange', updateCalendarEvents);
            AppStore.removeListener('currentScheduleIndexChange', updateCalendarEvents);
        };
    }, []);
```